### PR TITLE
Improve synths pages

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

File: src/xst.md [hunk 1]
```diff
@@ -73,9 +73,11 @@ Some more interesting details available from the [XSTUSD RFP](https://github.com
 
 The implementation of SORA Synthetics is just the beginning, another interesting implementation tied to this is the [SORA Social Insurance](social-insurance.md).
 
-In the meantime, you can find XSTUSD on [Polkaswap](https://polkaswap.io/). The XST asset id is `0x0200090000000000000000000000000000000000000000000000000000000000`
+In the meantime, you can find XSTUSD on [Polkaswap](https://polkaswap.io/). The XST asset id is `0x0200090000000000000000000000000000000000000000000000000000000000`. You can also access all synthetic assets on Polkaswap by visiting the [wallet page](https://polkaswap.io/#/wallet) and activating a `Show only synthetic tokens` flag in the user interface, similar to what's shown on the screen below:
 
-You can swap XOR to XSTUSD and vice-versa.
+![](/.gitbook/assets/polkaswap-show-only-synthetics.png)
+
+Consequently, you can swap those assets. You can swap XOR to XSTUSD and vice-versa like that:
```
Comment (WRRicht3r) [inline src/xst.md]:
```suggestion
Consequently, you can swap those assets. This is how to swap XOR to XSTUSD and vice-versa:
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

**Review by outoftardis — APPROVED** [review]
please address William's comment, and it's all good

File: src/xst.md [hunk 2]
```diff
@@ -73,9 +73,11 @@ Some more interesting details available from the [XSTUSD RFP](https://github.com
 
 The implementation of SORA Synthetics is just the beginning, another interesting implementation tied to this is the [SORA Social Insurance](social-insurance.md).
 
-In the meantime, you can find XSTUSD on [Polkaswap](https://polkaswap.io/). The XST asset id is `0x0200090000000000000000000000000000000000000000000000000000000000`
+In the meantime, you can find XSTUSD on [Polkaswap](https://polkaswap.io/). The XST asset id is `0x0200090000000000000000000000000000000000000000000000000000000000`. You can also access all synthetic assets on Polkaswap by visiting the [wallet page](https://polkaswap.io/#/wallet) and activating a `Show only synthetic tokens` flag in the user interface, similar to what's shown on the screen below:
```
Comment (WRRicht3r) [inline src/xst.md]:
```suggestion
In the meantime, you can find XSTUSD on [Polkaswap](https://polkaswap.io/). The XST asset id is `0x0200090000000000000000000000000000000000000000000000000000000000`. You can also access all synthetic assets on Polkaswap by visiting the [wallet page](https://polkaswap.io/#/wallet) and activating the `Show only synthetic tokens` flag in the user interface, as per the screen below:
```

File: src/xst.md [hunk 3]
```diff
@@ -73,9 +73,11 @@ Some more interesting details available from the [XSTUSD RFP](https://github.com
...
 
-You can swap XOR to XSTUSD and vice-versa.
+![](/.gitbook/assets/polkaswap-show-only-synthetics.png)
+
+Consequently, you can swap those assets. This is how to swap XOR to XSTUSD and vice-versa:
```
Comment (WRRicht3r) [inline src/xst.md]:
```suggestion
You can also swap those assets. This is how to swap XOR to XSTUSD and vice-versa:
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

**Review by WRRicht3r — APPROVED** [review]

**Pull request merged**